setup.py

```diff
@@ -42,10 +42,12 @@
             + "webviz_subsurface.plugins:ParameterParallelCoordinates",
             "RunningTimeAnalysisFMU = webviz_subsurface.plugins:RunningTimeAnalysisFMU",
             "RelativePermeability = webviz_subsurface.plugins:RelativePermeability",
+            "ReservoirSimulationTimeSeriesRegional = "
+            + "webviz_subsurface.plugins:ReservoirSimulationTimeSeriesRegional",
         ]
     },
     install_requires=[
-        "fmu-ensemble>=1.2.2",
+        "fmu-ensemble>=1.2.3",
```

Comment:
Is this a required bump wrt. this PR?

Reply:
`1.2.3` is the first version after https://github.com/equinor/fmu-ensemble/pull/134 where the`first` option used for `FOPT` `FGPT` and `FWPT` to check if the ensemble lacks history was introduced. Possible to bypass it by loading e.g. yearly instead and filter later on if you want to. `first` is simpler and stores less data though.